views/funding/create.php

```diff
@@ -17,22 +18,38 @@
 Select2BootstrapAsset::register($this);
 ?>
 
-<?php ModalDialog::begin(['header' => Yii::t('XcoinModule.base', 'Select wanted asset'), 'closable' => false]) ?>
+<?php ModalDialog::begin(['header' => Yii::t('XcoinModule.base', 'Select wanted asset and define exchange rate'), 'closable' => false]) ?>
 <?php $form = ActiveForm::begin(['id' => 'account-form']); ?>
 <?= Html::hiddenInput('step', '1'); ?>
 <div class="modal-body">
-    <?=
-    $form->field($model, 'asset_id')->widget(Select2::classname(), [
-        'data' => $assetList,
-        'options' => ['placeholder' => '- Select asset - ', 'value' => ($defaultAsset) ? $defaultAsset->id : []],
-        'theme' => Select2::THEME_BOOTSTRAP,
-        'hideSearch' => true,
-        'pluginOptions' => [
-            'allowClear' => false,
-            'escapeMarkup' => new JsExpression("function(m) { return m; }"),
-        ],
-    ]);
-    ?>
+    <?= $form->field($model, 'available_amount')->widget(AmountField::class, ['asset' => $myAsset])->label(Yii::t('XcoinModule.base', 'Maximum offered amount')); ?>
+    <p class='alert alert-info'>
+        The current balance of the funding account is: <strong><?= $fundingAccountBalance; ?></strong>
+    </p>
+    <hr/>
+    <p><?= Yii::t('XcoinModule.base', 'Determine the exchange rate for which you are willing to trade assets.'); ?></p>
+    <div class="row">
+        <div class="col-md-5">
+            <?= $form->field($model, 'exchange_rate')->widget(AmountField::class, ['asset' => $myAsset])->label(Yii::t('XcoinModule.base', 'Provided asset')); ?>
+        </div>
+        <div class="col-md-2 text-center">
+            <i class="fa fa-exchange colorSuccess" style="font-size:28px;padding-top:24px" aria-hidden="true"></i>
+        </div>
+        <div class="col-md-5">
+            <?=
+            $form->field($model, 'asset_id')->widget(Select2::classname(), [
+                'data' => $assetList,
+                'options' => ['placeholder' => '- Select asset - ', 'value' => ($defaultAsset) ? $defaultAsset->id : []],
```

Comment:
translate this: change to `['placeholder' => '- ' . Yii::t('XcoinModule.funding', 'Select asset') . ' - ', 'value' => ($defaultAsset) ? $defaultAsset->id : []],`

Reply:
I think we need to merge translation PR first , then handle the translation in this PR .